### PR TITLE
Design implementation of branding and keyword search

### PR DESCRIPTION
**Pull request by joshmcarthur**
This pull request compliments the last Android PR and begins to implement the branding and UI for the iOS app. This pull request is focussed around adding the top bar and keyword search. It's mostly done (enough to PR at least), but there are still some things that I'd like to care of that I'll enumerate here:

1. I'd like the search box to have a little more margin on the left and right - I've had a stab at this, but it's a fairly minor thing and I'm going to loop back and fix it up if I have time.
2. The segmented button in the top right corner is red as per designs - I had it red, but Nanz and Nani pretty much ruled it straight out - it's not quite right. I've made it white for now. I also think the hand image is grainier than it should be (maybe it's not using the high DPI image?)
3. I've not written tests for the search UI like I did for the Android - basically the same reason as the first point - I'm worried I'll accidentally sink a whole afternoon getting it going when it's not really that important as long as manual testing continues to be done. I'll loop back to it if I have time at the end (I think I will) - or do it on a Friday.

Screenshot time!

Here's the design:

screen shot 2017-06-07 at 10 59 20 am

And screenshots from the app:

simulator screen shot 8 06 2017 1 26 39 pm

simulator screen shot 8 06 2017 1 26 47 pm

simulator screen shot 8 06 2017 1 27 57 pm

### CODE REVIEW

**Comment by elspeth-rabid** [comment]
@joshmcarthur - I'm not sure about that blue used to border (any) in handshape and location.

**Comment by joshmcarthur** [comment]
@elspeth-rabid I totally get what you mean, and thought about changing it, but I actually have a story coming up to overhaul that type of search anyway, so I'm going to fix it next week.

File: NZSLDict/Classes/SearchViewController.swift [hunk 1]
```diff
@@ -252,7 +243,37 @@ class SearchViewController: UIViewController, UISearchBarDelegate, UITableViewDa
...
+        self.tabBarController?.navigationItem.setLeftBarButtonItems([
+            UIBarButtonItem.init(customView: navbarTitleFirstSegment),
+            UIBarButtonItem.init(customView: navbarTitleSecondSegment)
+        ], animated: false)
+        
+        
+        let navigationBarRightButtonItem = UIBarButtonItem.init(customView: modeSwitch);
+        self.tabBarController?.navigationItem.setRightBarButtonItem(navigationBarRightButtonItem, animated: false)
+        
         if wotdLabel.respondsToSelector("setAttributedText:") {
```
Comment (elspeth-rabid) [inline NZSLDict/Classes/SearchViewController.swift]: wotd? Typo or acronym?

File: NZSLDict/Classes/SearchViewController.swift [hunk 2]
```diff
@@ -178,7 +170,7 @@ class SearchViewController: UIViewController, UISearchBarDelegate, UITableViewDa
         wotdView = UIView(frame: searchTable.frame)
```
Comment (elspeth-rabid) [inline NZSLDict/Classes/SearchViewController.swift]: Ah not a typo.

**Pull request merged**